libs/components/src/next/cascader/vl-cascader.component.cy.ts

```diff
@@ -194,6 +194,22 @@ describe('component vl-cascader default', () => {
         cy.get('vl-cascader').invoke('attr', 'hide-breadcrumb', 'true');
         cy.get('vl-cascader').shadow().find('nav').should('not.exist');
     });
+
+    it('should display the subtitle', () => {
+        cy.get('vl-cascader')
+            .shadow()
+            .find('vl-cascader-item[label="West-Vlaanderen"]')
+            .shadow()
+            .find('vl-annotation')
+            .should('have.text', 'ondertitel');
+
+        cy.get('vl-cascader')
+            .shadow()
+            .find('vl-cascader-item[label="Oost-Vlaanderen"]')
+            .shadow()
+            .find('vl-annotation')
```

Comment:
Zoals in andere comment, deze test dan ook eventueel aanpassen - geen annotation > ook geen `vl-annotation` element.

Reply:
done